standardFormats/generic_roster_widget/roster.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "course_id": "100",
+    "course_code": "100A",
+    "name": "Example Course",
+    "course_link": "https://example.course.edu/info",
+    "schedule": [
+      {
+        "start": 1714115736753,
+        "end": 1714125836758,
+        "description": "Event description",
+        "location": "Campus, building, room number",
+        "summary": "Event A Title"
+      },
+      {
+        "start": 1714115736753,
+        "end": 1714125836758,
+        "recurrence":"Mo, We, Fr",
+        "until": 1714225836758,
+        "description": "Event description",
+        "location": "Campus, building, room number",
+        "summary": "Event B Title"
+      },
+      {
+        "start": 1714115736753,
+        "end": null,
+        "all_day":true,
+        "description": "Event description",
+        "location": "Campus, building, room number",
+        "summary": "Event C Title"
+      }
+    ],
+    "students": [
+      {
+        "name": "John Smith",
+        "user_id": "john.smith@example.edu"
+      }
+    ]
+  },
+  {
+    "user_id": "user.id@example.edu",
+    "courses": [
```

Comment:
Should the test files reference the only course that exists so that there is a relationship?

Reply:
That would make sense. Technically, it could be any course, as courses and user registrations can be sent independently or as an update etc. But for clarity, absolutely.

Reply:
Done